Sources/Purchasing/Purchases.swift

```diff
@@ -1632,194 +1637,49 @@ public extension Purchases {
-
-    /**
-     * Configures an instance of the Purchases SDK with a specified API key and app user ID.
-     *
-     * The instance will be set as a singleton.
-     * You should access the singleton instance using ``Purchases/shared``
-     *
-     * - Note: Best practice is to use a salted hash of your unique app user ids.
-     *
-     * - Warning: Use this initializer if you have your own user identifiers that you manage.
+     * - Note: Use this initializer if you wish to adjust network request timeouts.
      *
-     * - Parameter apiKey: The API Key generated for your app from https://app.revenuecat.com/
-     *
-     * - Parameter appUserID: The unique app user id for this user. This user id will allow users to share their
-     * purchases and subscriptions across devices. Pass `nil` or an empty string if you want ``Purchases``
-     * to generate this for you.
+     * - Parameter configuration: The ``Configuration`` object you wish to use to configure ``Purchases``
      *
      * - Returns: An instantiated ``Purchases`` object that has been set as a singleton.
-     */
-    @objc(configureWithAPIKey:appUserID:)
-    @discardableResult static func configure(withAPIKey apiKey: String, appUserID: String?) -> Purchases {
-        configure(withAPIKey: apiKey, appUserID: appUserID, observerMode: false)
-    }
-
-    /**
-     * Configures an instance of the Purchases SDK with a custom `UserDefaults`.
-     *
-     * Use this constructor if you want to
-     * sync status across a shared container, such as between a host app and an extension. The instance of the
-     * Purchases SDK will be set as a singleton.
-     * You should access the singleton instance using ``Purchases/shared``
-     *
-     * - Parameter apiKey: The API Key generated for your app from https://app.revenuecat.com/
      *
-     * - Parameter appUserID: The unique app user id for this user. This user id will allow users to share their
-     * purchases and subscriptions across devices. Pass `nil` or an empty string if you want ``Purchases``
-     * to generate this for you.
-     *
-     * - Parameter observerMode: Set this to `true` if you have your own IAP implementation and want to use only
-     * RevenueCat's backend. Default is `false`.
-     *
-     * - Returns: An instantiated ``Purchases`` object that has been set as a singleton.
+     * - Important: See ``Configuration/Builder`` for more information about configurable properties.
+     * 
      */
-    @objc(configureWithAPIKey:appUserID:observerMode:)
-    @discardableResult static func configure(withAPIKey apiKey: String,
-                                             appUserID: String?,
-                                             observerMode: Bool) -> Purchases {
-        configure(withAPIKey: apiKey, appUserID: appUserID, observerMode: observerMode, userDefaults: nil)
+    @objc(configureWithConfiguration:)
+    @discardableResult static func configure(with configuration: Configuration) -> Purchases {
```

Comment:
Yess so nice to have only one now 🕺🏻 

We'll we need to update the docs to reflect this?

Reply:
CC'ing @codykerns